classes/PodsInit.php

```diff
@@ -822,6 +824,28 @@ public function register_assets() {
 
 		wp_set_script_translations( 'pods-dfv', 'pods' );
 
+		$config = [
+			'wp_locale'      => $GLOBALS['wp_locale'],
+			'currencies'     => PodsField_Currency::$currencies,
```

Comment:
:warning: Array double arrow not aligned correctly; expected 1 space(s) between "'currencies'" and double arrow, but found 5.

<sub>hash: 0823a2a4ecb53a523839f958deb657a6</sub>